lib/v2/fzdm/manhua/manhua.js

```diff
@@ -0,0 +1,56 @@
+const got = require('@/utils/got');
+const domain = 'manhua.fffdm.com';
+const host = `https://${domain}`;
+const { art } = require('@/utils/render');
+const path = require('path');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const get_pic = async (url) => {
+    const response = await got(url);
+    const data = response.data;
+    return {
+        comicTitle: data.mhinfo.title,
+        chapterTitle: data.title,
+        pics: data.cont,
+        pubDate: timezone(parseDate(data.mh.time, 'YYYY-MM-DDTHH:mm:ss.SSSZ'), +8),
```

Comment:
```suggestion
        pubDate: parseDate(data.mh.time),
```
No need to adjust timezone for ISO 8601 timestring
[more info](https://stackoverflow.com/a/36798830)